.github/workflows/future-proof.yml

```diff
@@ -28,8 +28,10 @@ jobs:
         feature-nightly-dropck_eyepatch: ['', '--features nightly-dropck_eyepatch']
         include:
           # Also future-proof against semver breakage from dependencies.
-          - rust-toolchain: stable
+          - os: ubuntu-latest
```

Comment:
Should we include one of these jobs for each OS or would that just be overkill/completely unnecessary? I guess platform dependent dependencies could potentially break on their respective OS which is not detectable when building on a different OS or am I wrong in that assumption? Is the job even necessary given that `maybe-dangling` does not have any dependencies?

Reply:
Happy to delete this entire section instead, but perhaps @danielhenrymantilla had something in mind here.

Reply:
Good point; I think we can _comment it out_, with a "uncomment if and when dependencies are added to this project" slapped onto it 👍

Reply:
Commented it out.